source/agora/test/EnrollmentManager.d

```diff
@@ -96,3 +96,50 @@ unittest
                  node.getPreimage(enroll_3.utxo_key) != PreImageInfo.init,
             5.seconds));
 }
+
+// Test for restoring enrollments and pre-images from the Genesis block
+unittest
+{
+    import agora.consensus.validation.PreImage;
+    import core.thread;
+    import std.conv;
+
+    const validator_cycle = 20;
+    auto params = new immutable(ConsensusParams)(validator_cycle);
+    auto network = makeTestNetwork(TestConf.init, params);
+    scope(exit) network.shutdown();
+    scope(failure) network.printLogs();
+    network.start();
+    network.waitForDiscovery();
+
+    // Boilerplate
+    auto nodes = network.clients;
+    auto node_1 = nodes[0];
+
+    // Check if genesis block has enrollments
+    const b0 = node_1.getBlocksFrom(0, 2)[0];
+    assert(b0.header.enrollments.length == 4);
```

Comment:
```suggestion
    assert(b0.header.enrollments.length >= 1);
```

Since you are only using one, let's simplify this. This also ensures that if we extend the genesis block, this assert doesn't trigger needlessly.